views/search/_facets.gohtml

```diff
@@ -1,27 +1,64 @@
-<div class="bc-toolbar bc-toolbar--auto mb-4">
-    <div class="bc-toolbar-left">
-        <div class="bc-toolbar-item">
-            <div class="badge-list">
-                {{range .SearchArgs.Facets}}
-                {{partial (printf "search/%s_facet" (. | replace "." "_")) $}}
-                {{end}}
-            </div>
-        </div>
+<div
+  class="bc-toolbar bc-toolbar--auto mb-4{{if gt (len .SearchArgs.FacetLines) 2}} toggle-zone{{
+    end
+  }}"
+>
+  <div class="bc-toolbar-left">
+    <div class="bc-toolbar-item">
+      {{ range.SearchArgs.FacetLines }}
+      <div class="badge-list">
+        {{ range .}}
+        {{partial (printf "search/%s_facet" (. | replace "." "_")) $}}
+        {{ end }}
+      </div>
+      {{ end }}
     </div>
-    <div class="bc-toolbar-right">
-        <div class="bc-toolbar-item">
-            <div class="c-button-toolbar">
-                <form method="GET" action="{{.CurrentURL|queryClear}}">
-                    <input type="hidden" name="q" value="{{.SearchArgs.Query}}">
-                    {{range .SearchArgs.Sort}}
-                    <input type="hidden" name="sort" value="{{.}}">
-                    {{end}}
-                    <button class="btn btn-link btn-muted text-muted" type="submit">
-                        <i class="if if-close"></i>
-                        <span class="btn-text">Reset filters</span>
-                    </button>
-                </form>
-            </div>
+  </div>
+  <div class="bc-toolbar-right">
+    <div class="bc-toolbar-item">
+      <div class="c-button-toolbar">
+        {{if gt (len .SearchArgs.FacetLines) 2}}
```

Comment:
I created a constant const for this number of lines (2) [here](https://github.com/ugent-library/biblio-backoffice/blob/7f30bcb376ef31bb76d37533af9f866a06c3c422/models/search_args.go#L9) but not sure how to use that in gohtml. Probably would need to pass that on with the context?

Reply:
we clean that up after the templ rewrite